apps/src/bin/anoma-node/cli.rs

```diff
@@ -18,33 +18,65 @@ pub fn main() -> Result<()> {
             }
         },
         cmds::AnomaNode::Gossip(sub) => match sub {
-            cmds::Gossip::Run(cmds::GossipRun(args)) => {
-                let tx_source_address = ctx.get_opt(&args.tx_source_address);
-                let tx_signing_key = ctx.get_opt_cached(&args.tx_signing_key);
+            cmds::Gossip::Run(cmds::GossipRun(args::GossipRun {
+                addr,
+                // TODO: currently unused
```

Comment:
these args weren't being used, will open a follow-up

Reply:
coming to think of this, I think we can remove these as these options are now available via both the config and env vars

Reply:
done